lib/custom_widgets/popularity_adult_and_original_language.dart

```diff
@@ -0,0 +1,35 @@
+import 'package:flutter/material.dart';
+import 'middle_text.dart';
+
+class PopularityAdultAndLanguage extends StatelessWidget {
+  const PopularityAdultAndLanguage(
+      {super.key,
+      required this.popularity,
+      required this.adult,
+      required this.originalLanguage});
+
+  static const String language = 'Language:';
+  static const String staticPopularity = 'Popularity:';
+  static const String staticAdult = 'Adult:';
+  final num popularity;
```

Comment:
in relation to the previous comment , i would rename this pupularityValue

Reply:
done